packages/compass-assistant/src/prompts.ts

```diff
@@ -1,4 +1,6 @@
-export const buildExplainPlanPrompt = ({
+const explainPlanSystemPrompt = `TODO:....add custom prompt stuff here`;
```

Comment:
add these prompts in future PR.

julian or whoever is working on this can iterate on the prompts and PR them